content/rc/security/aws-transit-gateway.md

```diff
@@ -0,0 +1,137 @@
+
+    {{<image filename="images/rc/tgw-share-transit-gateway.png" width="80%" alt="The Share Transit Gateway section." >}}{{< /image >}}
+
+1. Follow the guide to [Update a resource share](https://docs.aws.amazon.com/ram/latest/userguide/working-with-sharing-update.html) in the [AWS resource access manager](https://console.aws.amazon.com/ram/). 
+
+    During the **Grant access to principals** step, select **AWS Account** in the **Select principal type** field. Enter the copied AWS account number in the **Enter an AWS Account ID** field. 
+
+    {{<image filename="images/rc/aws-tgw-add-principal.png" width="80%" alt="The AWS Add principal field." >}}{{< /image >}}
+
+    After the principal is added, it may take some time before it is associated. You can see the status of the principals under **Shared Principals** in the resource share page.
+
+#### AWS CLI
+
+To use the AWS CLI to set up the resource share:
+
+1. From the [Redis Cloud admin console](https://app.redislabs.com/), select the **Subscriptions** menu and then select your subscription from the list.
+
+1. Select **Connectivity > Transit Gateway** to view the transit gateway settings.
+
+1. In the **Share Transit Gateway** section, select **Copy** under **AWS CLI Command** to copy the Redis AWS Account number.
+
+    {{<image filename="images/rc/tgw-share-transit-gateway.png" width="80%" alt="The Share Transit Gateway section." >}}{{< /image >}}
+
+1. Enter the copied CLI command into a terminal shell. Replace `<TGW ARN>` with the Amazon resource name of your transit gateway.
+
+### Accept resource share and create attachment {#accept-resource-share}
+
+After you've associated the Redis AWS account with your resource share, you must accept the resource share in the admin console.
+
+1. In your Redis Cloud subscription's Transit Gateway settings, you should now see that a **Resource Share** is available. Select **Resource Shares** to view the resource share you initiated.
+
+    {{<image filename="images/rc/tgw-resource-shares-button.png" width="250px" alt="The Share Transit Gateway section." >}}{{< /image >}}
+
+1. Select **Accept** to associate the **Resource Share** with your admin console account.
+
+    {{<image filename="images/rc/tgw-accept-resource-shares.png" width="80%" alt="The Accept resource shares section." >}}{{< /image >}}
+
+1. Select **Close** to close the **Accept resource shares** section.
+
+1. You will now see your transit gateway in the **Transit Gateways** section. After the **TGW status** is **Available**, select **Create Attachment** under **Attachment status**. 
+
+    {{<image filename="images/rc/tgw-create-attachment-button.png" width="250px" alt="The Create attachment button." >}}{{< /image >}}
+
+    This will request an peering attachment representing Redis's AWS account to the Transit Gateway. After the attachment is created, it is in **Pending acceptance** status.
```

Comment:
@cmilesb the customer TGW has a configuration that either automatically accepts attachments or requires manual acceptance. Only in the latter case the attachment will reach the "pending acceptance" state.

Reply:
Makes sense. I'll change that, then